_docs/faq.markdown

```diff
@@ -3,17 +3,17 @@ title: FAQ
 order: 999999
 ---
 
-{% include h title="Where is Rust SDK?" tag="sdk" %}
+{% include h title="Where is the Rust SDK?" tag="sdk" %}
```

Comment:
I am never sure whether to use or not to use articles in titles, but I think they are rather omitted. Also personally it feels better to me without this `the` 😄